components/agentSearchForm/index.jsx

```diff
@@ -57,6 +72,16 @@ function AgentSearchForm({
       {invalidWebIdField ? (
         <Message variant="invalid">Please provide a valid WebID</Message>
       ) : null}
+      {isPodOwner ? (
```

Comment:
`isPodOwner && (<Message />)` is a slightly cleaner way to handle conditional rendering than a ternary.